src/screens/Server.js

```diff
@@ -30,12 +31,14 @@ class Server extends React.Component {
      * loading {boolean} If should show spinner while waiting for server response
      * newServer {boolean} If user selected checkbox that he wants to set a new server
      * selectedValue {string} Server selected from the user
+     * testnetServer {boolean} If user selected checkbox that he wants to connect to the testnet
```

Comment:
I'd call it mainnet. I feel that the most important thing is whether we are connected to the mainnet or not.

Reply:
Do you think the checkbox should be "I want to connect to the mainnet"? If that's what you mean by your comment I don't really agree.

The mainnet is the default network, thus if someone wants to deceive a user into connecting in the testnet to pretend that they received real tokens, I think the checkbox and the messages should be about the testnet, like I added "I want to connect to the testnet and understand that the tokens have no value there".

If we require the checkbox only for the mainnet someone might add a testnet server and won't need to read/select the checkbox.

What do you think?

Reply:
I meant just the flag. The checkbox should keep the testnet message. If the flag is `mainnetServer` and the checkbox is market, the flag is `false`.

Reply:
This code is outdated